### ISSUE TITLE
exclusion filter may not be working correctly on the "Fixed/Freely Moving Spiking Frequency Correlation" function? 

### ISSUE THREAD

**Issue by Sepidak** [RichardFav/AnalysisGUI#25]
some of those matched pairs that are included must have been filtered out by the exclusion filter.

So it seems that the drop down menu shows all those matched pairs that are also in Adam's dataframe regardless of filter. But the plots seem to show the numbers after filtering?  I'm fine with this arrangement as long as you confirm this is what's happening and the rest of this function works too. 

By the way, be ware that some cells are excluded in Adam's pipeline too because they have very low activity, so I don't expect all the matched pairs to appear in the drop down here, which is the case. 

to replicate results use CA514_A_day3.mcomp in Z4.
For the exclusion filter, exclude everything except RSPd and RSPg.

Note that in order to run this function I changed all "DARK1" keys to "DARK" in the code (both in main _analysis.py and in thread_workers.py, just to get passed one of the issues in #21, but I have now changed them back to the original (there are only 3).
This function seems to have multiple issues. Still even after running it, the individual cell correlations don't show up and the values are so low that I am not sure it's working correctly. But as long as it doesn't run smoothly I can't check to see whether the values are correct or not.

**Comment by RichardFav** [comment]
It turns out the code had indeed not been set up correctly to deal with the exclusion filter. This has been remedied and updated on Subiculum. I have checked this filter extensively and it appears to be correct, although you will need to conclusively test the filter so that it is correct (there _**could**_ be issues if the filter is too restrictive which will need to be dealt with further). 

**Note that the issue with "DARK"/"DARK1" keys has been resolved so it won't matter which key is used.**
